src/styles/nav.css

```diff
@@ -0,0 +1,119 @@
+/* ---- base ---- */
```

Comment:
We should namespace all selectors in this file under `.header-navigation` or something like that since we're targeting native elements and some pretty generic classes.